elasticsearch/Dockerfile

```diff
@@ -12,7 +12,7 @@ ENV HOME=/opt/app-root/src \
   SG_VER=2.4.1.8 \
   SG_SSL_VER=2.4.1.16 \
   ES_CLOUD_K8S_VER=2.4.1 \
-  OSE_ES_VER=2.4.1.5 \
+  OSE_ES_VER=2.4.1.6 \
```

Comment:
Update to https://github.com/fabric8io/openshift-elasticsearch-plugin/releases/tag/openshift-elasticsearch-plugin-2.4.1.7  .  I pushed 'release' on previous version before actually merging the dependent PR.